pages/etat.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import withData from '../lib/apollo/withData'
+import withT from '../lib/withT'
+
+import Frame from '../components/Frame'
+
+import { Editorial, Interaction, TitleBlock, A } from '@project-r/styleguide'
+
+import { PUBLIC_BASE_URL, STATIC_BASE_URL } from '../lib/constants'
+
+const { Lead, P, LI } = Editorial
+
+export default withData(
+  withT(({ url, t }) => {
+    const meta = {
+      pageTitle: t('etat/pageTitle'),
+      title: t('etat/title'),
+      description: t('etat/description'),
+      image: `${STATIC_BASE_URL}/static/team/bern.jpg`,
+      url: `${PUBLIC_BASE_URL}${url.pathname}`
+    }
+
+    return (
+      <Frame
+        url={url}
+        meta={meta}
+      >
+
```

Comment:
nit: two empty lines are too much IMHO. I'd even argue we don't need *any* empty lines to structure plain HTML/JSX markup, at least not between similar siblings like `<P>`s and `<LI>`s.